FV3/atmos_cubed_sphere/driver/fvGFS/atmosphere.F90

```diff
@@ -248,6 +248,8 @@ module atmosphere_mod
   integer :: sphum, liq_wat, rainwat, ice_wat, snowwat, graupel  ! condensate species tracer indices
 #ifdef CCPP
   integer :: cld_amt
+#else
+  !$ser verbatim integer::cld_amt
```

Comment:
Why have this inside the ifdef/else statement?

Reply:
I wasn't sure if it would mess up the CCPP mode to have it declared twice if there was CCPP and serialization